eng/common/cross/toolchain.cmake

```diff
@@ -47,6 +47,9 @@ elseif(TARGET_ARCH_NAME STREQUAL "arm64")
   if(TIZEN)
     set(TIZEN_TOOLCHAIN "aarch64-tizen-linux-gnu/9.2.0")
   endif()
+elseif(TARGET_ARCH_NAME STREQUAL "loongarch64")
```

Comment:
The changes in this file should be reverted per discussion in https://github.com/dotnet/arcade/pull/8271#discussion_r772107492

If this file needs more changes, they should be submitted via [arcade repo](https://github.com/dotnet/arcade/)

Reply:
Ok.
Thanks for your guidance.